docs/authoring.md

```diff
@@ -0,0 +1,93 @@
+# Authoring Components
+
+## Overview
+C#/WinRT provides support for authoring Windows Runtime components. You can write a library in C#, and use C#/WinRT's source generator to get a winmd that any WinRT compatible language can use. For example, a library written in C# can be used by a C++ program, via C#/WinRT and C++/WinRT, with just a few tweaks to the C++ project.
+
+
+## Authoring the C# Component
+To create a library, select the Class Library (.NET Core) template in Visual Studio. 
+Make sure the `<TargetFramework>` for the project is `net5.0-windows10.0.19041.0`. 

```

Comment:
For the TFM, it could also target net5.0-windows10.0.18362.0 and net5.0-windows10.0.17763.0 correct?

Reply:
Yeah that's true. How do we describe the need for specifying `net5.0-windows10.0.xxxxx.0` instead of just `net5.0` ?